packages/wdio-cli/src/constants.ts

```diff
@@ -284,6 +284,12 @@ export const QUESTIONNAIRE = [{
         : hasFile('tsconfig.json')
             ? COMPILER_OPTIONS.ts // default to TypeScript
             : COMPILER_OPTIONS.nil // default to no compiler
+}, {
+    type: 'confirm',
+    name: 'generateTSConfigFile',
+    message: 'Do you want WebdriverIO to autogenerate tsconfig file?',
```

Comment:
I wonder if this is the correct question to ask here. What happened if a user says `no`? They would see a bunch of TS errors due to a missing tsconfig, right?

Reply:
@christian-bromann  Yes thats why we added automatic creation of it ( in previous implementation ) . IN this case there is a console message saying what needs to be added to the tsconfig file so if user types  no still we are giving this console message so user have to add those to tsconfig file .

Reply:
Why do we need this question then?

Reply:
@christian-bromann  now we removed the automatic creation , if user says yes then we creates the file in current or test folder depending on availability of already existing tsconfig file. else we wont do anything . do you want to remove the question ?

Reply:
Yes, I think we clarified in https://github.com/webdriverio/webdriverio/issues/7640#issuecomment-972694037 that we always want to create a tsconfig file. Now I also suggest to put the tsconfig file always in `./test` because it simplifies the implementation and has no difference for the user. Then we don't need this question.

Reply:
@christian-bromann will do that 🥳

Reply:
Awesome, sorry if I wasn't clear enough at the beginning.